.github/workflows/sql-workbench-test-and-build-workflow.yml

```diff
@@ -1,6 +1,21 @@
 name: SQL Workbench Test and Build
 
-on: [pull_request, push]
+on:
+  pull_request:
+    types: [opened, reopened]
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+    paths:
+      - '**/*.java'
+      - '**/*.g4'
+      - '!sql-jdbc/**'
+      - '**gradle*'
+      - '**lombok*'
+      - '**/*.jar'
+      - '**/*.pom'
```

Comment:
i don't think these changes need to trigger workbench, since currently it only runs unit test not integration test?

Reply:
Thank you, fixed in 63c51377.